### PR TITLE
Added Secure Templates for 'Automation Account variables must be encrypted' control

### PR DESCRIPTION
**Pull request by saurabhgautam-msft**

### CODE REVIEW

File: 08-Secure by default/Secure Templates/Bicep/AutomationAccounts.bicep [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+@description('Specify the name of automation account\'s variable')
+param automationAccountVariableName string
+
+@description('Specify the value of automation account\'s variable')
+param automationAccountVariableValue string
+
+@description('Specify the description of automation account\'s variable')
+param automationAccountVariableDescription string
+
+resource automationAccount 'Microsoft.Automation/automationAccounts@2020-01-13-preview' = {
```
Comment (Aboli-msft) [inline 08-Secure by default/Secure Templates/Bicep/AutomationAccounts.bicep:16]: Any particular reason for using this preview API? I see updated and stable version available:  api-version=2021-06-22

File: 08-Secure by default/Secure Templates/Bicep/AutomationAccounts.bicep [hunk 2]
```diff
@@ -0,0 +1,37 @@
+@description('Specify the name of automation account to be created.\r\nUse only alphanumerics and hyphens.\r\nThe name must not start or end with alphanumeric and must be 6 - 50 characters in length.\r\n')
+param automationAccountName string
```
Comment (vaishnavipulluri) [inline 08-Secure by default/Secure Templates/Bicep/AutomationAccounts.bicep:2]: This bicep script is able to create automation account with only one variable right. Did we think of supporting multiple variables with account creation or just add variables for the existing account?

Comment (saurabhgautam-msft) [reply 08-Secure by default/Secure Templates/Bicep/AutomationAccounts.bicep:2]: Yeah, so the expectation from the template is to provide the valid property value. And even one type of variable is sufficient to convey this.

File: 08-Secure by default/Secure Templates/ARM/AutomationAccounts.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            "type": "string",
+            "metadata": {
+            "description": "Specify the description of automation account's variable"
+            }
+        }
+    },
+    "resources": [
+      {
+        "type": "Microsoft.Automation/automationAccounts",
+        "apiVersion": "2020-01-13-preview",
```
[the remark this answers is not in the document]
Comment (saurabhgautam-msft) [reply 08-Secure by default/Secure Templates/ARM/AutomationAccounts.json]: No there is no particular reason,

I have updated the version to 22-08-08

File: 08-Secure by default/Secure Templates/Bicep/AutomationAccounts.bicep (hunk 1 above)
Comment (saurabhgautam-msft) [reply 08-Secure by default/Secure Templates/Bicep/AutomationAccounts.bicep]: No there is no particular reason,

I have updated the version to 22-08-08

**Review by Aboli-msft — APPROVED** [review]

**Review by TarunKrShukla — APPROVED** [review]

**Pull request merged**